spec/controllers/api/dashboards_get_spec.rb

```diff
@@ -496,5 +496,28 @@
       expect(response.status).to eq(400)
       expect(data).to eq({errors: [{status: 400, title: "Invalid page size"}]})
     end
+
+    it 'includes meta and links objects with extra info for pagination' do
+      get :index
+
+      body = json_response
+
+      expect(body).to include(:data)
+      expect(body[:data]).to be_a(Array)
+
+      expect(body).to include(:links)
+      expect(body[:links]).to be_a(Object)
+      expect(body[:links][:self]).to be_a(String)
+      expect(body[:links][:first]).to be_a(String)
+      expect(body[:links][:last]).to be_a(String)
+      expect(body[:links][:prev]).to be_nil
```

Comment:
`prev` =  `self`on the first page.

Reply:
In theory, Ruby's gem will_paginate respects JSON API spec, but in this specific case, when on first page, prev = null. So prev = self will never hold true, despite in theory it should.

After Slack chat, decided to keep it like this.